pkg/acsengine/defaults.go

```diff
@@ -582,10 +582,26 @@ func setMasterProfileDefaults(a *api.Properties, isUpgrade bool) {
 	if a.MasterProfile.HTTPSourceAddressPrefix == "" {
 		a.MasterProfile.HTTPSourceAddressPrefix = "*"
 	}
+	// Set VMSS Defaults for Masters
```

Comment:
Is there a reason we're not doing this in a func like the vmss for agents defaults flow above?